open_event/static/js/admin/event/new.js

```diff
@@ -0,0 +1,75 @@
+$( document ).ready(function() {
+    $('.add-session-type').click(function (){
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Block must be padded by blank lines.](https://www.codacy.com/app/triplez/open-event-orga-server/file/2650524462/issues/source?bid=3233864&fileBranchId=3324174#l2)